docs/API.md

```diff
@@ -276,7 +276,7 @@ Parameters.
 Each token takes the following fields for it's create method respectively:
 
 * sessionToken : data, uid, createdAt, uaBrowser, uaBrowserVersion, uaOS, uaOSVersion, uaDeviceType,
-                 tokenVerificationId
+                 mustVerify, tokenVerificationId
```

Comment:
Also in `fxa-auth-db-server/docs/API.md` circa lines 878 and 941, it needs adding to the documented responses of the endpoints too.